procedures/MT/m2/m2_control_slew.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>use margin instead of space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Fixed in the comment: [1405646](https://github.com/lsst-ts/ts_notebooks/pull/111/commits/1405646fa99724e333a1234bbb5ccaf972337eb6). Thanks!